components/Events/EventsItemCard.vue

```diff
@@ -56,6 +58,7 @@ interface Props {
 
 withDefaults(defineProps<Props>(), {
   types: () => [],
+  altText: "Event or seminar image",
```

Comment:
A seminar is a type of event, you I would just say 
```suggestion
  altText: "Event image",
```

Reply:
I chose this since this is the default behavior, and later on the majority of events are treated differently if they are a regular event or seminar, so this kind of acted as a checkpoint before the separation of the two. I can change it if you like, but to me this way was more consistant

Reply:
That's a good point, I'm changing it 👍